package/standalone_file_logger/standalone_file_logger/src/rotating_logger.cc

```diff
@@ -187,35 +187,65 @@ double RotatingLogger::get_time_passed()
 
 void RotatingLogger::frame_handler(const uint8_t *data, size_t size)
 {
-  if (!_dest_available) {
-    // check imediately on startup for path availability. Subsequently, check
-    // periodically
-    if (_session_start_time.time_since_epoch().count() != 0 && get_time_passed() < _poll_period) {
-      return;
+  // TODO: keep track of total data bytes: enforce a limit/warning in .h
+  std::vector<uint8_t> *frame = new std::vector<uint8_t> (data, data + size);
```

Comment:
I don't think a vector<uint8_t> is a good fit here - an array<uint8_t> would be better?

Reply:
I used `vector` here to avoid having to also pass around the size information. (`unique_ptr<uint8_t[]>` no longer tracks size). Do you expect there to be a notable performance / memory hit?

Reply:
a byte per vector element definitely seems pretty bloaty - the std::array<uint8t_> seems like a good fit as its meant to handle fixed sizes.